packages/ui-components/src/components/button.tsx

```diff
@@ -3,6 +3,10 @@
  * Distributed under the terms of the Modified BSD License.
  */
 
+/*
+ * @deprecated You should use the `Button` component from `@jupyter/react-components` instead.
```

Comment:
Could you move this line at line 33 (within the doc string of `Button`)? This will highlight the deprecation within the code in advanced editors.